crates/testing-framework/src/transform/parser.rs

```diff
@@ -239,7 +239,7 @@ fn parse_sexp_call_triplet(inp: Input<'_>) -> IResult<Input<'_>, Box<Triplet>, P
             delimited(
                 delim_ws(tag("(")),
                 separated_pair(
-                    context("triplet service name", parse_sexp_string),
+                    context("triplet service name has to be a string", parse_sexp_string),
```

Comment:
Do you mean a literal string or scalar is also fine as a module name?

Reply:
If literal is the only option I would change it to `has to be a literal string`.

Reply:
It is only literal string that can be used as a service ID, you are right.